tests/dogtag/pytest-ansible/pytest/performance_test/test_cert_revocation.py

```diff
@@ -98,11 +98,11 @@ def run(self):
         client.join()
 
     end = timer()
-
+    print(end)
```

Comment:
Is this a debug statement? Do we need to keep this?

Reply:
Yes, I removed all that but miss these two.